js/locales/udata.en.json

```diff
@@ -123,6 +123,13 @@
     "Drag a picture here": "Drag a picture here",
     "Drop resource": "Drop resource",
     "Edit": "Edit",
+    "Edit dataset \"{title}\"": "Edit dataset \"{title}\"",
```

Comment:
I would use single quotes for the outside text in this context for avoiding the escaping.